ikasaneip/visualisation/dashboard/src/main/java/org/ikasan/dashboard/security/ContextCache.java

```diff
@@ -0,0 +1,20 @@
+package org.ikasan.dashboard.security;
+
+import com.vaadin.flow.component.UI;
+
+import java.util.HashMap;
+
+public class ContextCache {
```

Comment:
@mick-stewart73 

this cash will never expire ? This might cause memory leak.

Is the COntexCache utilised by user session or dashboard backend?

Reply:
Good spot @andrzej-majewski. I have changed this to a cache and set an expiry. Unfortunately we cannot put these values into the session.